tests/buffer/buffer_api_common.h

```diff
@@ -287,6 +290,7 @@ void test_buffer(util::logger &log, sycl::range<dims> &r,
            "sycl::buffer::get_count() does not return "
            "the correct number of elements");
     }
+#endif
```

Comment:
```suggestion
#endif  // SYCL_CTS_ENABLE_DEPRECATED_FEATURES_TESTS
```

Makes it easier to track down the beginning and end